packages/@atjson/document/test/annotation-test.ts

```diff
@@ -0,0 +1,80 @@
+import TestSource, { Bold } from './test-source';
+
+describe('Document#annotations', () => {
```

Comment:
This isn't actually testing `Document#annotations`, but instead `Annotation#equals`. There's a test file already for annotations that this may be best for.